.github/workflows/style_check.yml

```diff
@@ -0,0 +1,47 @@
+name: Style check
+
+on:
+  # Trigger on pull requests.
+  pull_request:
+
+  # Trigger on pushes to the mainline branches and version tags. This prevents building commits
```

Comment:
Does the version tags bit need to be specified below? Right now I only see the branch trigger.

Reply:
This is a miscopied comment. There is no need to run style checks on tags.